tools/applyformatpatch.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+rm -f patch.diff
```

Comment:
You should check if this file exists first and  print a small message informing about its deletion